docs/unified/novacustom/recovery.md

```diff
@@ -81,102 +141,288 @@ make BOARD=arduino/mega2560
+1. Pry the keyboard away from the laptop. Use a plastic spudger to release the
+   tabs holding it in place, starting from the top.
 
-- Disconnect the keyboard from the laptop
+    !!! warning
 
-=== "NS5x / NS7x"
-    ![](/images/nvc_ec_flash/ns5x_keyboard_removed.jpg)
+        Be extra careful when removing the keyboard to avoid damaging the fragile
+        keyboard cable!
 
-=== "NV4x"
-    ![](/images/nvc_ec_flash/nv4x_keyboard_removed.jpg)
+1. Unplug the keyboard connector by lifting up the tab holding it in place:
 
-- Connect the USB-C cable to the Thunderbolt port on the laptop and to the host
-  computer. This will provide grounding
-- Connect the FFC cable to the FFC breakout board
+    ![](/images/nvc_ec_flash/ns5x_keyboard_connectors.jpg)
+
+1. Connect the FFC cable to the FFC breakout board
 
-=== "NS5x / NS7x"
     ![](/images/nvc_ec_flash/ns5x_arduino_breakout.jpg)
-    > Note: In the example above, the FFC connector on the breakout has the pins
-    > on the **bottom** side of the connector and is located on the **same**
-    > side as the pins connecting to the Arduino. If your breakout is different,
-    > you may need an FFC cable with connectors on the **opposite** sides.
 
-=== "NV4x"
-    ![](/images/nvc_ec_flash/nv4x_arduino_breakout.jpg)
-    > Note: In the example above, the FFC connector on the breakout has the pins
-    > on the **upper** side of the connector and is located on the side
-    > **opposite** of the pins connecting to the Arduino. If your breakout
-    > is different, you may need an FFC cable with connectors on the
-    > **opposite** sides.
+    !!! warning
 
-- Insert the breakout into Arduino's digital pin header, pins 22-45, with the
-  FFC connector facing outwards
+        In the example above, the FFC connector on the breakout has the pins on
+        the **bottom** side of the connector and is located on the **same** side
+        as the pins connecting to the Arduino. If your breakout is different,
+        you may need an FFC cable with connectors on the **opposite** sides.
 
-  ![](/images/nvc_ec_flash/ns5x_arduino_breakout_attached.jpg)
+1. Insert the breakout into Arduino's digital pin header, pins 22-45, with the
+   FFC connector facing outwards
 
-- Connect the other end of the FFC cable to the keyboard connector on the
-  laptop, taking care to align pin 1 of the FFC cable to pin 1 (leftmost) pin
-  of the connector
+    ![](/images/nvc_ec_flash/ns5x_arduino_breakout_attached.jpg)
+
+1. Connect the other end of the FFC cable to the keyboard connector on the
+   laptop, taking care to align pin 1 of the FFC cable to pin 1 (leftmost) pin
+   of the connector
 
-=== "NS5x / NS7x"
     ![](/images/nvc_ec_flash/ns5x_arduino_connected.jpg)
 
-=== "NV4x"
-    ![](/images/nvc_ec_flash/nv4x_arduino_connected.jpg)
+1. Connect the Arduino to the host using the USB-A to USB-B cable
+1. Connect the USB-C cable together with power blocker from your host
+   computer to the laptop.
```

Comment:
It only worked after I'd plugged the battery in for the flashing operation, as @mkopec suggested. Would be useful to add that to the list

Reply:
@dawidlesiak please add this to docs